### PR TITLE
Harpy Instrument Additions

### PR DESCRIPTION
**Pull request by cynical24**
# Description

Adds three new harpy instruments: Piano, Church Organ, and Harp.

---

# Changelog

:cl:
- add: three new harpy instruments

### CODE REVIEW

File: Content.Server/Instruments/SwappableInstrumentComponent.cs [hunk 1]
```diff
@@ -11,4 +11,11 @@ public sealed partial class SwappableInstrumentComponent : Component
     /// </summary>
     [DataField("instrumentList", required: true)]
     public Dictionary<string, (byte, byte)> InstrumentList = new();
+
+    // Frontier: harpy instruments
+    /// <summary>
+    /// When true, only the instrument entity itself can swap its sound.
+    /// </summary>
+    [DataField]
+    public bool OnlySetBySelf = true;
```
Comment (Mnemotechnician) [inline Content.Server/Instruments/SwappableInstrumentComponent.cs:20]: This is true by default, meaning that ALL swappable instruments are affected by this change. You didn't modify the yaml prototypes of other instruments, meaning that all swappable instruments (synthesizers, guitars, etc) are affected by this change and cannot be swapped by anyone unless they get possessed by an admin...

You should make it false by default, and set it to true on MapInit in SingerSystem instead.

**Review by Mnemotechnician — CHANGES_REQUESTED** [review]

File: Content.Server/Instruments/SwappableInstrumentComponent.cs (hunk 1 above)
Comment (cynical24) [reply Content.Server/Instruments/SwappableInstrumentComponent.cs:20]: okie dokie

**Comment by Fansana** [comment]
just some general advice, try writing commit messages that are a bit more on topic, later on that might help to narrow down where an important change happend.

**Comment by Memeji** [comment]
What's the status on this PR?

**Review by FoxxoTrystan — APPROVED** [review]

**Review by ShatteredSwords — APPROVED** [review]
All 3 instruments work after playesting

**Pull request merged**